src/sections/fragen.html

```diff
@@ -0,0 +1,84 @@
+
+    </div>
+    <div class="space-y-44">
+        <div class="mb-24">
+            <div class="space-x-3 text-sm">
+                <div class="grid grid-cols-1 md:grid-cols-2 gap-24 mt-24 lg:pr-28 lg:pl-80 font-montserrat-bold gap-y-24">
+                    <div class="flex flex-col">
+                        <div class="flex justify-between px-5">
+                            <p>Welche Frage steht hier?</p>
+                            <p class="text-xl font-montserrat-alternates-bold">+</p>
+                        </div>
+                        <div class="w-full h-line bg-green-light"></div>
+                    </div>
+                    <div class="flex flex-col">
+                        <div class="flex justify-between px-5">
+                            <p>Welche Frage steht hier?</p>
+                            <p class="text-xl font-montserrat-alternates-bold">+</p>
+                        </div>
+                        <div class="w-full h-line bg-green-light"></div>
+                    </div>
+                    <div class="flex flex-col">
+                        <div class="flex justify-between px-5">
+                            <p>Welche Frage steht hier?</p>
+                            <p class="text-xl font-montserrat-alternates-bold">+</p>
+                        </div>
+                        <div class="w-full h-line bg-green-light"></div>
+                    </div>
+                    <div class="flex flex-col">
+                        <div class="flex justify-between px-5">
+                            <p>Welche Frage steht hier?</p>
+                            <p class="text-xl font-montserrat-alternates-bold">+</p>
+                        </div>
+                        <div class="w-full h-line bg-green-light"></div>
+                    </div>
+                    <div class="flex flex-col">
+                        <div class="flex justify-between px-5">
+                            <p>Welche Frage steht hier?</p>
+                            <p class="text-xl font-montserrat-alternates-bold">+</p>
+                        </div>
+                        <div class="w-full h-line bg-green-light"></div>
+                    </div>
+                    <div class="flex flex-col">
+                        <div class="flex justify-between px-5">
+                            <p>Welche Frage steht hier?</p>
+                            <p class="text-xl font-montserrat-alternates-bold">+</p>
+                        </div>
+                        <div class="w-full h-line bg-green-light"></div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="space-y-12">
+            <div>
+                <p class="font-bold text-lg">Nehmen Sie Kontakt zu uns auf!</p>
+            </div>
+            <div>
+                <span class="border flex items-center justify-center sm:inline bg-yellow-500 text-black rounded-full px-4 py-1 sm:py-3 gap-4 mt-">
```

Comment:
Remove unnecessary styles like this one.
```suggestion
                <span class="border flex items-center justify-center sm:inline bg-yellow-500 text-black rounded-full px-4 py-1 sm:py-3 gap-4">
```